src/components/StatusBadge/index.tsx

```diff
@@ -23,6 +23,7 @@ interface StatusBadgeProps {
   serviceUrl?: string;
   tmdbId?: number;
   mediaType?: 'movie' | 'tv';
+  enableTooltip?: boolean;
```

Comment:
As discussed via DMs, we shouldn't have to add this parameter. The issue is that the case where `serviceUrl` is `null` is not handled properly, and we can resolve this by fixing the `else if` clause.